docs/pages/components/tile.md

```diff
@@ -917,4 +917,474 @@ In action mode view, the close button is displayed on the top right-hand corner
+        <ul class="docs-ul">
+            <li>Line Tile - apply the <code>fd-tile--line</code> modifier class to <code>fd-tile</code>.
+                Line Tiles have two modes - cosy (default) and compact (<code>fd-tile--compact</code>). *Note: the compact mode is applied only on Line Tiles. It should not be applied on the other types.
+                <ul class="docs-ul">
+                    <li>Line Tile Header - <code>fd-tile__header</code>
+                        <ul class="docs-ul">
+                            <li>Title <code>fd-tile__title</code> - <strong>required</strong>. For a Line Tile with a badge the title and the badge must be wrapped in a container <code>fd-tile__title-container</code>.
+                            </li>
+                            <li>Subtitle <code>fd-tile__subtitle</code> <strong>optional</strong>.</li>
+                        </ul>
+                    </li>
+                </ul>
+            </li>
+        </ul>
+    </li>
+</ul>
+
+<br>
+<strong>Breakpoints</strong>
+<ul class="docs-ul">
+    <li><strong>List</strong> behaviour - for screen sizes less than 450px - S screens.</li>
+    <li><strong>Floating</strong> behaviour - for screen sizes greater or equal to 450px - M, L and XL screens.</li>
+</ul>
+
+<br>
+<strong>Wrapping and Overflow Behaviour</strong>
+<ul class="docs-ul">
+    <li><strong>Floating</strong> behaviour - when next link doesn't fit right to previous link, it gets dropped in the next row underneath. If the content of the Tile can't fit in one line, the Tile wraps into two rows. *Note: this behaviour requires JavaScript and is not provided in Fundamental-Styles. </li>
+    <li><strong>List</strong> behaviour - the Tile grows vertically and the text inside wraps into two rows, splitting title in the first row and subtitle in the second row.</li>
+</ul>
+
+<br>
+<strong>List Tile States</strong>
+<ul class="docs-ul">
+    <li>Regular</li>
+    <li>Hover</li>
+    <li>Pressed</li>
+    <li>Disabled - the disabled state can only be applied on Line Tiles. The other types do not have a disabled state. </li>
+</ul>
+
+<br>
+<strong>Action Mode</strong> <br><br>
+The controls are wrapped in a container <code>fd-tile__action-container</code> which is right aligned within the tile. There's a 0.25rem spacing between the header and the actions container. On edit mode List View the overflow of the list is done with gradient mask.
+
+{% capture tile-line %}
+<h4>Line Tile - Floating Behaviour, Cosy Mode</h4>
+<div class="docs-section-container">
+    <div class="fd-tile-container"> 
+        <div tabindex="0" class="fd-tile fd-tile--line">
```

Comment:
Every `fd-tile` in this file should have a role of `button` as they are clickable items
UI5 also does this: https://sapui5.hana.ondemand.com/#/entity/sap.m.GenericTile/sample/sap.m.sample.GenericTileAsLaunchTile

Everything else looks good to me.

Reply:
Thank you @jacobdevera  I added role button to every tile